playbooks/common/openshift-cluster/upgrades/v3_1_to_v3_2/node_upgrade.yml

```diff
@@ -0,0 +1,24 @@
+- name: Prepare for Node evacuation
+  command: >
+    {{ openshift.common.admin_binary }} manage-node {{ openshift.common.hostname | lower }} --schedulable=false
+  delegate_to: "{{ groups.oo_first_master.0 }}"
```

Comment:
This came out as /usr/local/bin/oadm in an rpm install, which does not exist. I am confused as to why that fact is wrong.